src/main/kotlin/com/zepben/evolve/services/network/translator/NetworkCimToProto.kt

```diff
@@ -966,9 +1010,27 @@ fun toPb(cim: Recloser, pb: PBRecloser.Builder): PBRecloser.Builder =
 fun toPb(cim: RegulatingCondEq, pb: PBRegulatingCondEq.Builder): PBRegulatingCondEq.Builder =
     pb.apply {
         controlEnabled = cim.controlEnabled
+        cim.regulatingControl?.also { regulatingControlMRID = it.mRID } ?: clearRegulatingControlMRID()
```

Comment:
Preferring also when I don't need to return whatever happens inside the block. I'm probably not being consistent everywhere because I keep forgetting, but Anthony gave me the heads up a while back and it seemed to make sense, he had a good reason at the time but I don't quite remember why. Something about accidentally returning things you weren't meant to with let.